src/som/Output.java

```diff
@@ -0,0 +1,82 @@
+package som;
+
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+
+import som.vm.VmSettings;
+
+
+public class Output {
+
+  public static class AnsiColor8 {
+    public static final String BLACK   = "\u001b[30m";
+    public static final String RED     = "\u001b[31m";
+    public static final String GREEN   = "\u001b[32m";
+    public static final String YELLOW  = "\u001b[33m";
+    public static final String BLUE    = "\u001b[34m";
+    public static final String MAGENTA = "\u001b[35m";
+    public static final String CYAN    = "\u001b[36m";
+    public static final String WHITE   = "\u001b[37m";
+    public static final String RESET   = "\u001b[0m";
+  }
+
+  @TruffleBoundary
+  public static void print(final String msg, final String color) {
+    // Checkstyle: stop
+    if (VmSettings.NO_ANSI_COLOR_IN_OUTPUT) {
+      System.out.print(msg);
+    } else {
+      System.out.print(color + msg + AnsiColor8.RESET);
+    }
+    // Checkstyle: resume
+  }
+
+  @TruffleBoundary
+  public static void println(final String msg, final String color) {
+    print(msg + System.lineSeparator(), color);
+  }
+
+  @TruffleBoundary
+  public static void print(final String msg) {
+    print(msg, AnsiColor8.WHITE);
+  }
+
+  @TruffleBoundary
+  public static void println(final String msg) {
+    println(msg, AnsiColor8.WHITE);
+  }
+
+  @TruffleBoundary
+  public static void errorPrintln() {
+    println("", AnsiColor8.RED);
+  }
+
+  @TruffleBoundary
+  public static void errorPrintln(final String msg) {
+    println(msg, AnsiColor8.RED);
+  }
+
+  @TruffleBoundary
+  public static void errorPrint(final String msg) {
+    print(msg, AnsiColor8.RED);
+  }
+
+  @TruffleBoundary
+  public static void warningPrintln(final String msg) {
+    println(msg, AnsiColor8.YELLOW);
+  }
+
+  @TruffleBoundary
+  public static void warningPrint(final String msg) {
+    print(msg, AnsiColor8.YELLOW);
+  }
+
+  @TruffleBoundary
+  /**
+   * This method is used to print reports about the number of created artifacts.
+   * For example actors, messages and promises.
+   */
```

Comment:
Please first the comment, then the annotation.

Reply:
https://github.com/smarr/SOMns/blob/master/src/som/VM.java#L352

Reply:
Yeah, there should be a linter rule for that...